integration/__init__.py

```diff
@@ -0,0 +1,23 @@
+from marionette import Marionette, MarionetteTestCase
+from firefox_puppeteer.testcases import BaseFirefoxTestCase
+
+
+class FirefoxTestCase(BaseFirefoxTestCase, MarionetteTestCase):
+
+    # HACK: Replicate setUp from superclass, but just without the navigation to
+    # browser.newtab.url
+    def setUp(self, *args, **kwargs):
```

Comment:
Hmm, yeah, I think I'm just using whatever the default is. I'll add that to this base test case though, for our tests